api/controllers/ParserController.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  list: function(req, res) {
+    var limit = Utils.parseLimit(req),
+      offset = Utils.parseSkip(req),
+      parser_version = req.param('parser_version');
+
+    sequelize.query(`SELECT p.name as name, pv.version as version FROM DownloadStatistics ds, Packages p, PackageVersions pv, ParsingJobs job
```

Comment:
This query should be in the models, see others models to see how to add custom queries to it, 
I would even put it in the Packages model, as the main entity that you are retrieving is a Packages.
You can then call Packages.findOldUnfailedPackage or whatever you call it directly from here.
That keeps the database logic out of the controllers